Extension.swift

```diff
@@ -25,6 +25,8 @@ extension Color {
     //etc
     static let background = Color("background")
     static let cdplayer = Color("cdplayer")
+    static let darkbg = Color("darkbg")
```

Comment:
Camel case 를 지키는게 좋을 것 같아요 ~ 
그렇게 길지 않아서, 축약하지 않고 이름을 사용하는건 어떨까요 ?

```suggestion
    static let darkBackground = Color("darkbg")
```